jsonrpc/utils/test/payloadId.test.ts

```diff
@@ -46,4 +42,67 @@ describe("Payload Id", () => {
     const duplicates = findDuplicates(results);
     chai.expect(duplicates.length === 0).to.be.true;
   });
+
+  it("generates increasing values when called within same tick", () => {
+    let i = 0;
+    while (i++ < 10000) {
+      const value1 = payloadId();
+      const value2 = payloadId();
+      const value3 = payloadId();
+      const value4 = payloadId();
+      if (value1 >= value2 || value2 >= value3 || value3 >= value4) {
+        chai.assert.fail("Not increasing values");
+      }
+    }
+    chai.assert.isOk("Pass");
+  });
+
+  it("generates non-repeating values when called within same tick", () => {
+    let i = 0;
+    while (i++ < 10000) {
+      const values = [payloadId(), payloadId(), payloadId(), payloadId()];
+      const set = new Set(values);
+      if (set.size !== values.length) {
+        chai.assert.fail("Not unique values");
+      }
+    }
+    chai.assert.isOk("Pass");
+  });
+});
+
+describe("Get BigInt Rpc Id", () => {
+  it("returns a bigint", () => {
+    const value = getBigIntRpcId();
+    chai.expect(typeof value === "bigint").to.be.true;
+  });
+
+  it("returns a bigint with 23 digits", () => {
+    chai.expect(getBigIntRpcId().toString().length).to.equal(23);
```

Comment:
```suggestion
  it("returns a bigint with 19 digits", () => {
    chai.expect(getBigIntRpcId().toString().length).to.equal(19);
```
updated to 19 digits as explained in the above comment